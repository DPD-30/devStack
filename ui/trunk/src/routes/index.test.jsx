import React from 'react'
import { BrowserRouter as Router } from 'react-router-dom'
import { renderToString } from 'react-dom/server'


import RoutingIndex from './index.jsx'

jest.unmock('./index.jsx')


test('RoutingIndex matches snapshot', () => {
	const comp = renderToString(<Router><RoutingIndex /></Router>)

	expect(comp).toMatchSnapshot()
})
