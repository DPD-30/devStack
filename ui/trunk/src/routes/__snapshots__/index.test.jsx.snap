// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoutingIndex matches snapshot 1`] = `
<div>
  mock routing public
</div>
<div>
  mock routing azure user
</div>
<div>
  mock routing local user
</div>
<div>
  mock routing templates
</div>
`;
